frontend/src/metabase/visualizations/echarts/cartesian/model/other-series.ts

```diff
@@ -80,7 +84,7 @@ export const createOtherGroupSeriesModel = (
 
 export const getAggregatedOtherSeriesValue = (
   seriesModels: SeriesModel[],
-  aggregationType: AggregationType = "sum",
+  aggregationType: VisualizationSettings["graph.other_category_aggregation_fn"] = "sum",
```

Comment:
Can we use a nice type like `AggregationType` as it was before because this is a bit verbose?